README.md

```diff
@@ -1,4 +1,16 @@
-# gamesproject
+# The Space Invaders Project
 
-This is a work in progress on my final project to **CS50’s Web Programming with Python and JavaScript**
-this README will be updated as i go along..
+[![Front End Tests Actions Status](https://github.com/AvshRLev/gamesproject/workflows/Front%20End%20Tests/badge.svg)](https://github.com/AvshRLev/gamesproject/actions) [![Lint Code Base Action Actions Status](https://github.com/AvshRLev/gamesproject/workflows/Lint%20Code%20Base%20Action/badge.svg)](https://github.com/AvshRLev/gamesproject/actions) [![Website space-invaders.avshi.net](https://img.shields.io/website-up-down-green-red/https/space-invaders.avshi.net.svg)](https://space-invaders.avshi.net/)
+
+This is a project created end to end by me for learning and showcasing my abilities.
+To view the project online follow [this link](https://space-invaders.avshi.net/, "Link to my project online")
+
+## Table of contents
+
+1. project architecture
```

Comment:
each one of those should be a link to a header (h1) and under that header will be the full explanation